src/sandstorm/run-bundle.c++

```diff
@@ -1719,7 +1781,8 @@ private:
             true));
       }
 
-      KJ_SYSCALL(setenv("PORT", kj::str(config.port).cStr(), true));
+      // TODO: ",".join(config.ports) rather than relying on default behavior.
```

Comment:
You can use: `kj::strArray(config.ports, ",")`

However, this is the well-defined default behavior of `kj::str()` on an array, so IMO what you have is fine.